test/functional/eval/system_spec.lua

```diff
@@ -178,7 +178,7 @@ describe('system()', function()
 
   describe('passing no input', function()
     it('returns the program output', function()
-      eq("echoed", eval('system("echo -n echoed")'))
+      eq("echoed\n", eval('system("echo echoed")'))
```

Comment:
The `-n` was to avoid a newline. Try printf instead, it's POSIX-specified and therefore much more reliable that echo (which we really shouldn't be using anyway).


    eq("echoed", eval('system("printf echoed")'))